### PR TITLE
zh-cn(translate): block-level-content

### PR DESCRIPTION
**Pull request by fwqaaq**

### CODE REVIEW

File: files/zh-cn/glossary/block-level_content/index.md [hunk 1]
```diff
@@ -1,92 +1,43 @@
...
-### HTML
+> **备注：** HTML（_超文本标记语言_）元素历来被归类为“块级”元素或者“行级”元素。作为一种呈现的特性，现在这将由 CSS 来提供。
 
-```html
-<p>This paragraph is a block-level element; its background has been colored to display the paragraph's parent element.</p>
-```
+## 示例
 
-### CSS
+在这个示例中，两个段度（{{HTMLElement("p")}}）元素被放置在 {{HTMLElement("div")}} 中。
```
Comment (jasonren0403) [inline files/zh-cn/glossary/block-level_content/index.md:16]:
```suggestion
在这个示例中，两个段落（{{HTMLElement("p")}}）元素被放置在 {{HTMLElement("div")}} 中。
```

File: files/zh-cn/glossary/block-level_content/index.md [hunk 2]
```diff
@@ -1,92 +1,43 @@
...
 ```
 
-{{ EmbedLiveSample('块级元素示例') }}
-
-## 用法
-
-- 块级元素只能出现在 {{ HTMLElement("body") }} 元素内。
-
-## 块级元素与行内元素
+段度（{{HTMLElement("p")}}）元素默认是块级的。这是为什么它们显示在块布局中：
```
Comment (jasonren0403) [inline files/zh-cn/glossary/block-level_content/index.md:28]: 之前好像没有把 paragraph element 翻成“段度”过？且这两句话感觉读起来不太通顺

```suggestion
段落（{{HTMLElement("p")}}）元素默认是块级的。这是为什么它们显示在块布局中：
```

**Review by jasonren0403 — COMMENTED** [review]
some comments

File: files/zh-cn/glossary/block-level_content/index.md [hunk 3]
```diff
@@ -1,92 +1,43 @@
...
+在 CSS 中，参与行内布局的内容被称为**块级内容**（block-level content）。
 
-通常浏览器会在块级元素前后另起一个新行。下面的例子表明了块级元素的作用：
+在块级布局中，从包含块的顶部开始，盒子总是一个接着一个地垂直放置。每个盒子的左外边缘触及包含块的左边缘。块级元素总是开始在新的行/列上。在水平书写模式中，像英语或者阿拉伯语，它占用它父母元素（容器）的整个水平空间以及垂直空间等同于它的内容的高度，从而创建一个“区块”。
 
-## 块级元素示例
+> **备注：** 如果包含块的 [`writing-mode`](/zh-CN/docs/Web/CSS/writing-mode) 被设置为[默认值](/zh-CN/docs/Web/CSS/writing-mode#形式定义)之外的值，上述的块布局行为将发生改变。
 
-### HTML
+> **备注：** HTML（_超文本标记语言_）元素历来被归类为“块级”元素或者“行级”元素。作为一种呈现的特性，现在这将由 CSS 来提供。
```
Comment (yin1999) [inline files/zh-cn/glossary/block-level_content/index.md:12]: https://github.com/mdn/content/blob/ae36b1dcba1ff033983c58116d3df1dd72164ab6/files/jsondata/L10n-SVG.json#L413-L421

```suggestion
> **备注：** HTML（_超文本标记语言_）元素历来被归类为“块级”元素或者“行级”元素。作为一种表现特性，现在这将由 CSS 来提供。
```

File: files/zh-cn/glossary/block-level_content/index.md [hunk 4]
```diff
@@ -1,92 +1,43 @@
...
 ```
 
-{{ EmbedLiveSample('块级元素示例') }}
-
-## 用法
-
-- 块级元素只能出现在 {{ HTMLElement("body") }} 元素内。
-
-## 块级元素与行内元素
+段度（{{HTMLElement("p")}}）元素默认是块级的。这是为什么它们显示在块布局中：
```
Comment (yin1999) [inline files/zh-cn/glossary/block-level_content/index.md:28]:
```suggestion
段度（{{HTMLElement("p")}}）元素默认是块级的。这是它们显示在块布局中的原因：
```

**Review by yin1999 — COMMENTED** [review]
考虑一下这三处

File: files/zh-cn/glossary/block-level_content/index.md (hunk 4 above)
Comment (fwqaaq) [reply files/zh-cn/glossary/block-level_content/index.md:28]:
```suggestion
段落（{{HTMLElement("p")}}）元素默认是块级的。这是它们显示在块布局中的原因：
```

**Review by yin1999 — APPROVED** [review]

**Pull request merged**